src/pages/settings/Profile/Contacts/NewContactMethodPage.tsx

```diff
@@ -109,74 +107,72 @@ function NewContactMethodPage({route}: NewContactMethodPageProps) {
         Navigation.goBack(ROUTES.SETTINGS_CONTACT_METHODS.getRoute(navigateBackTo));
     }, [navigateBackTo]);
 
-    if (isActingAsDelegate) {
-        return <NotFoundPage onBackButtonPress={onBackButtonPress} />;
-    }
-
     return (
-        <ScreenWrapper
-            onEntryTransitionEnd={() => loginInputRef.current?.focus()}
-            includeSafeAreaPaddingBottom={false}
-            shouldEnableMaxHeight
-            testID={NewContactMethodPage.displayName}
-        >
-            <HeaderWithBackButton
-                title={translate('contacts.newContactMethod')}
-                onBackButtonPress={onBackButtonPress}
-            />
-            <FormProvider
-                formID={ONYXKEYS.FORMS.NEW_CONTACT_METHOD_FORM}
-                validate={validate}
-                onSubmit={handleValidateMagicCode}
-                submitButtonText={translate('common.add')}
-                style={[styles.flexGrow1, styles.mh5]}
+        <DelegateNoAccessWrapper accessDeniedVariants={[CONST.DELEGATE.DENIED_ACCESS_VARIANTS.DELEGATE]}>
```

Comment:
`DelegateNoAccessWrapper` is wrapped within `ScreenWrapper` everywhere else except here. Any reason for this?

Reply:
It is a mistake, It should be inside `ScreenWrapper`. Fixing...